metrics/storage/fio-k8s/fio-test-ci.sh

```diff
@@ -40,11 +40,11 @@ function main() {
 	metrics_json_start_array
 	local json="$(cat << EOF
 	{
-		"read-io": {
+		"readio": {
```

Comment:
Any specific reason for this change?

Reply:
the issue is that when I am reading with jq, I am having a lot of issues trying to read with -